bugbug/model.py

```diff
@@ -104,7 +104,7 @@ def bugs():
             print(metrics.confusion_matrix(y_test_filter, y_pred_filter, labels=class_names))
             print(classification_report_imbalanced(y_test_filter, y_pred_filter, labels=class_names))
 
-        joblib.dump(self, f'{self.__class__.__name__.lower()}')
+        joblib.dump(self.clf, self.__class__.__name__.lower())
```

Comment:
Why storing only the classifier?

Reply:
So that It will help us to predict the classes next time without any training. What would you suggest, which value to store in a dump?

Reply:
This is already the case right now, we are storing `self`. `clf` is a property of self, so we are storing `clf` too :)

Reply:
Thanks a lot, I have changed my code and committed the same.